scout/commands/serve.py

```diff
@@ -22,7 +23,8 @@ def serve(host, port, debug, livereload, test):
     """Start the web server."""
 
     # Verify the database connectivity before launching the app
-    mongo_client = current_app.config.get("MONGO_DATABASE")._Database__client
+    mongo_client: MongoClient = current_app.config.get("MONGO_DATABASE").client
```

Comment:
This code is supported also in Python 3.9. tested.

Reply:
I'm not the biggest fan of type hints in code, but as it is fairly deeply hidden in that call I guess it's ok. Also as you say it doesn't hurt, at least later Python versions. Using `.client` instead of the internal `_` one seems very prudent! 😁

Reply:
I've added the type hint because I was a bit confused myself, what was a database and what was a client etc. it doesn't hurt to keep it, for maintenance.